tests/Microsoft.Identity.Test.Android.UIAutomation/androidTests.cs

```diff
@@ -62,8 +62,12 @@ public void RunAllTests()
                 AcquireTokenSilentTest,
 
                 AcquireTokenADFSV3InteractiveFederatedTest,
+                AcquireTokenADFSV3InteractiveManagedTest,
+                AcquireTokenMsaUserTest,
                 AcquireTokenADFSV4InteractiveFederatedTest,
+                AcquireTokenADFSV4InteractiveManagedTest,
                 AcquireTokenADFSV2019InteractiveFederatedTest,
+                AcquireTokenADFSV2019InteractiveManagedTest,
```

Comment:
adfs2019 supports 2 modes: federated (i.e. through AAD) or direct. There is no such thing as "managed"

Reply:
adfsv3 and adfsv4 are supported via federation only, MSAL cannot "directly" talk to them.